filewriter.md

```diff
@@ -10,11 +10,11 @@ This method creates an object for the file library.
 ~~~
 
 ## fileWriter.writeToFile
-This method writes the specified data to a file in txt/doc format. On successful write, a url is returned which points to the location of file on server
+This method writes the specified data to a file in txt/doc format. On successful write, an URL is returned, which points to the server file's location.
```

Comment:
`an URL` should be `a URL`
`server file's location` should be `file's location on the server`